src/main/java/org/shanerx/tradeshop/listeners/ShopTradeListener.java

```diff
@@ -166,13 +166,18 @@ public void onBlockInteract(PlayerInteractEvent e) {
                 return;
         }
 
+        String owner = shop.getOwner().getName();
+
+        if (owner == null)
+            owner = "-Unknown-";
```

Comment:
Is `owner` ever expected to be null or is this just an instance of defensive programming?

Reply:
It is never expected and should never happen, however, I ran into on Dark Chromas test server. Hence the fix.

Reply:
Hmm, that’s odd…

Reply:
It could be that the servers player data was not copied over to their dev server so the players name didn't exist.